packages/components/psammead-topic-tags/README.md

```diff
@@ -0,0 +1,56 @@
+
+## Description
+
+The `TopicTags` component is a styled `<ul>` or `<div>`, depending on the number of `TopicTag` components it contains. If there is only a single `TopicTag` component with the `TopicTags` component, then the `TopicTags` component is a styled `<div>`, otherwise it's a styled `<ul>`.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-topic-tags --save
+```
+
+## Props
+
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| script | string | Yes | N/A | `'news'` |
+| service | object | Yes | N/A | `{ canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }` |
+| dir | string | No | `'ltr'` | One of `'ltr'` or `'rtl'` |
+
+## Usage
+
+<!-- Description of the component usage -->
+
+```jsx
+import { TopicTags, TopicTag } from "@bbc/psammead-topic-tags"
+import { latin } from '@bbc/gel-foundations/scripts';
+
+const Wrapper = () => (
+    <TopicTags script={latin} service="news" dir="ltr" >
+        <TopicTag topicName="Retailing" topicLink="/url/to/topic" />
+        <TopicTag topicName="Business" topicLink="/url/to/topic" />
+        <TopicTag topicName="Viruses" topicLink="/url/to/topic" />
+    </TopicTags>
+);
+```
+
+### When to use this component
+
+The `TopicTag` component should only be used inside of a `TopicTags` component, and a `TopicTags` component should only be used to contain `TopicTag` components.
+
+### Accessibility notes
+
+The `TopicTags` component is fundamentally a `<ul>`, and so the `TopicsTags` component is announced as a list when using a screen reader, using the ARIA `role="text"` attribute.
```

Comment:
Think you got slightly mixed up, it's role list to announce as a list, so: "...using the ARIA role="list" attribute, this reinstates the list semantics on VoiceOver which removes them due to the use of list style type none in the CSS."